src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  display: flex;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin-top: 32px;
+  }
+
+  &__model {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $font-main-color;
+    height: 36px;
+    width: 166px;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &__description {
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
This is default value for font-weight, so we don't have to write it
```suggestion
```

Reply:
Must I convert scss to css?